tests/common.go

```diff
@@ -9369,3 +9425,31 @@ func AddCloudCredentialOwnership(cloudCredentialName string, cloudCredentialUid
 	}
 	return nil
 }
+
+// GenerateS3BucketPolicy Generates an S3 bucket policy based on encryption policy provided
+func GenerateS3BucketPolicy(sid string, encryptionPolicy string, bucketName string) string {
+
+	encryptionPolicyValues := strings.Split(encryptionPolicy, "=")
```

Comment:
Please add a length check here, else the program may panic if incorrect input without `=` is provided. And let the function return an error along with the string.

Reply:
done